### DISCUSSION TITLE
How to monitor specific dockets for updates in real-time? when latency is of utmost importance

### DISCUSSION THREAD

**Discussion by codestoned1** [freelawproject/courtlistener#3002] [Q&A] [answered]
I want to track a few specific dockets on a very high frequency basis, using the minimum code required. How should I best accomplish this? I also saw Slack mentioned in the developer guide, but can't find an invite to it - any chance that is still around? Thanks!

**Comment by mlissner** [comment]
Let's connect via email and we can hash this out. Thanks for getting in touch!

**Comment by codestoned1** [comment, marked as answer]
thanks for the reply! emailed you earlier today :)

**Comment by legaltextai** [comment]
Just saw this thread and am also interested in monitoring dockets. Is there a way to set up RSS feed to see what's been submitted by any party?

**Reply by mlissner** [reply]
Every docket has an RSS feed, yes. See the information on the docket feeds, here: https://www.courtlistener.com/feeds/
